src/snark/libsnark/algebra/fields/tests/test_bigint.cpp

```diff
@@ -13,7 +13,7 @@ using namespace libsnark;
 
 TEST(algebra, bigint)
 {
-    static_assert(ULONG_MAX == 0xFFFFFFFFFFFFFFFFul, "unsigned long not 64-bit");
+    static_assert(UINT64_MAX == 0xFFFFFFFFFFFFFFFFul, "uint64_t not 64-bit");
```

Comment:
You might think this is true by definition, but notice the type of the constant. In order for this to do the right thing, ``unsigned long`` must be able to represent the integer 2<sup>64</sup>-1; if it can't, sensible compilers will give an error, but I'm not sure the behaviour is actually defined. In any case, that's not what we intended to check. (It should be perfectly fine if ``unsigned long`` *can't* represent 2<sup>64</sup>-1; that's the point of changing the types.) Delete the assertion!